core/src/main/java/com/valtech/aemsaas/core/models/search/SearchTemplate.java

```diff
@@ -0,0 +1,18 @@
+package com.valtech.aemsaas.core.models.search;
+
+import lombok.Getter;
+
+public enum SearchTemplate {
+  PRODS("prods"),
```

Comment:
those are justr sample values. we need to make this configurable. probably also a context aware configuration would make sense here so you can change on the fly